src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs

```diff
@@ -117,7 +117,7 @@ public static bool IsServerDomainJoined()
         // Returns 'true' if the client is running on a Windows OS
         public static bool IsWindows()
         {
-            return RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            return Platform.MatchesCurrentPlatform(PlatformID.Windows);
```

Comment:
I'm fine with code consistency being a reason to change this. I was wondering if there was some subtlety to comparing things differently that caused a slightly different behavior.